app/lib/ctc_question_navigation.rb

```diff
@@ -7,6 +7,8 @@ class CtcQuestionNavigation
     Ctc::Questions::ContactPreferenceController,
     Ctc::Questions::CellPhoneNumberController,
     Ctc::Questions::EmailAddressController,
+    # VerifyIdentityController goes here
```

Comment:
You can create a placeholder VerificationController here with the form_class defined as NullForm here, if you'd like. There won't be any naming interference because i haven't created the controller yet -- im still working on modifying the backend logic.